src/server/index.jsx

```diff
@@ -223,7 +251,6 @@ server.get(
         headers,
       });
 
-      // Return an internal server error for any uncaught errors
       res.status(500).send(message);
```

Comment:
We should consider only returning the error message if we're running locally - descriptive error messages being shown to the public can be embarrassing and a potential security issue

Reply:
Yeah I'd definitely hide the error message in production for security reasons primarly.

Reply:
We can add that to the refactor ticket 👍